test/test_blocks.py

```diff
@@ -216,6 +216,17 @@ def test_initiated_analytics(self, mock_post, monkeypatch):
             pass
         mock_post.assert_called_once()
 
+    def test_launch_analytics_does_not_error_with_invalid_blocks(self, monkeypatch):
+        monkeypatch.setenv("GRADIO_ANALYTICS_ENABLED", "True")
```

Comment:
You may want to monkeypatch the `_do_analytics_request` function to be a no-op though so this doesn't actually send spurious analytics from every CI run.